pkg/builder/quarkus.go

```diff
@@ -206,6 +207,7 @@ func GenerateQuarkusProjectCommon(runtimeVersion string, quarkusVersion string,
 				},
 			},
 		},
+		jib.JibMavenPlugin(),
```

Comment:
We should do this one only when the publish strategy is JIB. However, in general, I think we should leverage the builder profile configuration to have a clean logic separation (ie, the quarkus trait should know nothing about the publishing strategy).

Reply:
With the proposed approach, you should be able to create a configmap on the fly containing the required configuration for Jib and set to the build accordingly.